files/brews/zlib.rb

```diff
@@ -1,19 +1,62 @@
-require 'formula'
-
 class Zlibphp < Formula
-  homepage 'http://www.zlib.net/'
-  url 'http://zlib.net/zlib-1.2.8.tar.gz'
-  sha1 'a4d316c404ff54ca545ea71a27af7dbc29817088'
+  desc "General-purpose lossless data-compression library"
+  homepage "http://www.zlib.net/"
+  url "http://zlib.net/zlib-1.2.8.tar.gz"
+  mirror "https://downloads.sourceforge.net/project/libpng/zlib/1.2.8/zlib-1.2.8.tar.gz"
+  sha256 "36658cb768a54c1d4dec43c3116c27ed893e88b02ecfcb44f2166f9c0b7f2a0d"
 
-  keg_only :provided_by_osx
+  bottle do
+    cellar :any
+    revision 1
+    sha256 "2971abbd45572722af5043a74ecf8a5bfd06adc9834ec90e4126a34c6ce982a1" => :yosemite
+    sha256 "adc394a9e296003bc2bc88451c649aec019496cb6ed3d6673005fcd818ae44a5" => :mavericks
+    sha256 "6ece1cb4b656f0f7ef1feab95ef6eb183e2f9ee2448c1e034de1a97d7f9da249" => :mountain_lion
+  end
 
   version '1.2.8-boxen1'
```

Comment:
The `-boxen1` version number is just book keeping. There are NOT any boxen specific changes. It is the same URL as the base `1.2.8` version. The puppet-php just added the post fix for book keeping, so leaving it